### ISSUE TITLE
SSGTS --remediate-using causes test scenario errors

### ISSUE THREAD

**Issue by mildas** [ComplianceAsCode/content#8024]
#### Description of problem:
When `--remediate-using` option is used, SSGTS final phase results in error because it can't download ARF file from VM.

#### SCAP Security Guide Version:
883760448

#### Operating System Version:
RHEL9

#### Steps to Reproduce:

1. `$ python3 tests/test_suite.py rule --libvirt qemu:///session test-suite-rhel9 --datastream build/ssg-rhel9-ds.xml --scenarios service_disabled --remediate-using bash service_pcscd_enabled`

#### Actual Results:
```
Setting console output to log level INFO                                                                               
INFO - The base image option has not been specified, choosing libvirt-based test environment.                                                                                                                                                 
INFO - Logging into /home/mlysonek/SCAP/content/logs/rule-custom-2022-01-04-1200/test_suite.log                                                                                                                                               
INFO - xccdf_org.ssgproject.content_rule_service_pcscd_enabled                                                                                                                                                                                
INFO - Script service_disabled.fail.sh using profile (all) OK                                                          
ERROR - oscap info /home/mlysonek/SCAP/content/logs/rule-custom-2022-01-04-1200/service_pcscd_enabled-service_disabled.fail.sh-remediation-arf.xml returned 1 exit code
ERROR - Failed to copy root@192.168.122.8:/xccdf_org.ssgproject.content_rule_service_pcscd_enabled.sh to /home/mlysonek/SCAP/content/logs/rule-custom-2022-01-04-1200: 'scp' command returned non-zero.
ERROR - Failed to download file /xccdf_org.ssgproject.content_rule_service_pcscd_enabled.sh                                                                                                                                                   
ERROR - The remediation failed for rule 'xccdf_org.ssgproject.content_rule_service_pcscd_enabled'.                                                                                                                              
```

#### Expected Results:
No error

#### Additional Information/Debugging Steps:
Without `--remediate-using`, it's fine:
```
$ python3 tests/test_suite.py rule --libvirt qemu:///session test-suite-rhel9 --datastream build/ssg-rhel9-ds.xml --scenarios service_disabled service_pcscd_enabled                                                 
                                                           
Setting console output to log level INFO                                                                               
INFO - The base image option has not been specified, choosing libvirt-based test environment.                                                                                                                                                 
INFO - Logging into /home/mlysonek/SCAP/content/logs/rule-custom-2022-01-04-1158/test_suite.log                                                                                                                                               
INFO - xccdf_org.ssgproject.content_rule_service_pcscd_enabled                                                                                                                                                                                
INFO - Script service_disabled.fail.sh using profile (all) OK     
```

**Comment by matejak** [comment]
Damn, another issue caused by #7750, I will take a look into it.

**Comment by matejak** [comment]
Should be fixed by #8025

**Comment by mildas** [comment]
I believe this has been fixed. Closing
